### ISSUE TITLE
Feature Request: Pick the base raspbian image from local file system

### ISSUE THREAD

**Issue by youngsoul** [davidferguson/pibakery-raspbian#26]
I would like to be able to pick the version of raspbian to one I have downloaded previously.  Is it possible to specify the version somehow?

Thank you - great tool!

**Comment by youngsoul** [comment]
after looking at your implementation, I see that you have to create a custom raspbian image with your scripts installed to make this work.  Therefore, picking a generic version of raspbian is not possible - but I would still like to select from the various PiBakery versions.

**Comment by youngsoul** [comment]
Could you describe the steps, or do you have a script, to take the vanilla raspbian image, and then update the necessary components.  I would like to be able to create my own image that is 'PiBakery' ready

**Comment by yombo** [comment]
Here's what I came up with. Take the base raspbian/jessie image from raspberry pi website. Then either mount this locally using a loopback or copy to sd card, boot, and then run the below script.

The below assumes you copied to sdcard and running on the raspberry pi.  You will then need to copy the image back to a linux box and shrink the image. This allows you to tweak the image, remove/install software, do whatever you want. I highly suggest removing wolfram and/or libre office if you don't plan to use them. This process is documented here: http://www.aoakley.com/articles/2015-10-09-resizing-sd-images.php

**Tip**: Here a couple lines to remove 1.3 GB of bloat. New image size: 2.74GB, 721MB compressed.

```
sudo apt-get remove --purge libreoffice-* bluej wolfram-engine oracle-java8-jdk minecraft-pi python-minecraftpi python3-numpy omxplayer sonic-pi scratch nuscratch timidity smartsim claws-mail -y
sudo apt-get autoremove -y
sudo apt-get clean
sudo rm /usr/share/raspi-ui-overrides/applications/wolfram-*
```

I'll be releasing a new version of PiBakery with updated raspbian/jessie with a lot of the bloat removed in the next day. I'm stuck on getting the shell image to work, only the PIXEL image is working. See: https://github.com/davidferguson/pibakery/issues/87

```
#!/usr/bin/env bash
cd /tmp
wget https://github.com/dalmago/pibakery-raspbian/archive/master.zip
unzip master.zip
cd pibakery-raspbian-master
cp -d etc/systemd/system/multi-user.target.wants/* /etc/systemd/system/multi-user.target.wants/
cp -d etc/systemd/system/* /etc/systemd/system/
cp -d lib/systemd/system/* /lib/systemd/system/
cp -a opt/* /opt/
```

**Comment by davidferguson** [comment]
> I see that you have to create a custom raspbian image with your scripts installed to make this work. Therefore, picking a generic version of raspbian is not possible

This is true _currently_, but in a future version you'll actually be able to select any Raspberry Pi image from your computer, and PiBakery will be able to use that! Because of schoolwork and exams, I've not had time to quite finish this version yet, but look out for it coming in (hopefully) February!

**Comment by Timon0x31** [comment]
Ok, it's a year and a half later. Can we select an image rather than having one included in PiBakery? I'd much rather get my OSs from the foundation and not in some other program.
